operations/Makefile

```diff
@@ -1,19 +1,26 @@
 SHELL := bash
 .SHELLFLAGS := -eu -o pipefail -c
 
+# Pretty colors
+COLOR_PURPLE = \033[1;35m
```

Comment:
a vote to change this to `\033[1;44m` (which will make this show up as bold white on an ... ahum ... Azure background
we can call it `COLOR_AZURE`

Reply:
Fixed in next push